tests/foreman/cli/test_product.py

```diff
@@ -85,7 +85,7 @@ def test_positive_create_3(self):
         @Assert: Product is created and has random description
 
         """
-        for desc in valid_labels_list():
+        for desc in valid_data_list():
```

Comment:
The description test should use valid_data_list and not valid_labels_list.  I found this when searching for the usage of valid_labels_list in robottelo.